main.go

```diff
@@ -81,13 +79,13 @@ func newManager() (ctrl.Manager, error) {
 		options, ramenConfig = controllers.LoadControllerConfig(configFile, scheme, setupLog)
 	}
 
-	controllerType = ramenConfig.RamenControllerType
-	if !(controllerType == ramendrv1alpha1.DRCluster || controllerType == ramendrv1alpha1.DRHub) {
+	controllers.ControllerType = ramenConfig.RamenControllerType
+	if !(controllers.ControllerType == ramendrv1alpha1.DRCluster || controllers.ControllerType == ramendrv1alpha1.DRHub) {
 		return nil, fmt.Errorf("invalid controller type specified (%s), should be one of [%s|%s]",
-			controllerType, ramendrv1alpha1.DRHub, ramendrv1alpha1.DRCluster)
+			controllers.ControllerType, ramendrv1alpha1.DRHub, ramendrv1alpha1.DRCluster)
 	}
 
-	if controllerType == ramendrv1alpha1.DRHub {
+	if controllers.ControllerType == ramendrv1alpha1.DRHub {
```

Comment:
(nit) could define a `controllers.ConfigMapName` variable and set it in this if/else to avoid the if on ramenconfig.go line 266

Reply:
Yes we could but given that we already have a global variable ControllerType in main.go, I don't want to pollute more than necessary the global space.